### DISCUSSION TITLE
Sanity "(reading 'call')" error

### DISCUSSION THREAD

**Discussion by Nuf1i** [renovatebot/renovate#19497] [Request Help] [answered]
Hi I cloned Sanity project v3
and install package.json dependencies then when I run it on localhost the error was 
TypeError: Cannot read properties of undefined (reading 'call')

what is the problem ?

**Comment by rarkins** [comment, marked as answer]
I think you have the wrong project ?

**Reply by Nuf1i** [reply]
This project
https://github.com/sanity-io/nextjs-blog-cms-sanity-v3

**Reply by rarkins** [reply]
That's a different project to this one, you are in the wrong place
